reference/7.1/Microsoft.PowerShell.Core/About/about_Automatic_Variables.md

```diff
@@ -484,7 +484,7 @@ objects in a pipeline.
 
 ### $PSScriptRoot
 
-Contains the directory from which a script is being run.
+Contains the full path of the executing script's directory.
```

Comment:
just getting granular here but what do you think about:
```suggestion
Contains the full path of the executing script's parent directory.
```

I do think your version is much clearer than the original so I'm good with it.